app/scripts/controllers/utils/utils.test.js

```diff
@@ -0,0 +1,92 @@
+import { strict as assert } from 'assert';
```

Comment:
The tests are depend on the actual production block list, which may change for reasons.
Please use a mock `SNAP_BLOCKLIST`.

Example:
```javascript
function makeCheckBlockList(blockList) {
  return async function checkSnapsBlockList(snapsToCheck() {
    // ...
  }
}
```

Reply:
I've did couple of changes to achieve having this using mocked version properly. Now blocklist is mocked for each test.